conf.toml

```diff
@@ -43,3 +43,7 @@ HostOrchestratorPort = 1081
 
 [WebRTC]
 STUNServers = ["stun:stun.l.google.com:19302"]
+
+# Regex patterns of allowed origins
+# e.g. "https://localhost:[0-9]+"
+CorsAllowedOrigins = []
```

Comment:
Let's move the config parameter to the general section to the top below `WebStaticFilesPath` as this one is not a `WebRTC` specific configuration paremeter. In the future, we might add a new config group to list general web related config concepts.

Reply:
Moved CORSAllowedOrigins config from WebRTC section to the top level.